rollup.yaml

```diff
@@ -362,8 +362,9 @@ components:
           description: Unix timestamp of block in milliseconds.
           example: 1588598533000
         prev_randao:
-          type: integer
-          format: uint64
+          type: string
+          format: hex
+          pattern: "^0x([0-9a-fA-F]{2})*$"
```

Comment:
I would be more strict and say this is `^0x([0-9a-fA-F]{64})$`

Reply:
@guidanoli Updated!